.coderabbit.yaml

```
@@ -0,0 +1,27 @@
+    - "**/*.java"
+    - "**/*.yml"
+    - "**/*.xml"
+    - "**/*.vue"
+    - "**/*.js"
+
+# Configure review settings
+settings:
+  # Enable automatic reviews on pull requests
+  auto_review: true
+  # Require approval from CodeRabbit before merging
+  require_approval: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)